enclave-tls/src/crypto_wrappers/openssl/main.c

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2020-2021 Alibaba Cloud and Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <enclave-tls/crypto_wrapper.h>
+#include <enclave-tls/log.h>
+#include <enclave-tls/cert.h>
+
+crypto_wrapper_err_t openssl_pre_init(void);
+crypto_wrapper_err_t openssl_init(crypto_wrapper_ctx_t *ctx);
+crypto_wrapper_err_t openssl_gen_privkey(crypto_wrapper_ctx_t *ctx,
+				enclave_tls_cert_algo_t algo,
+				uint8_t *privkey_buf, unsigned int *privkey_len);
```

Comment:
use clang-format-9 available in ic dev image or ubuntu 18.04 to fix these indent issues